Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+const path = require( "path" );
+const loadGruntConfig = require( "load-grunt-config" );
+
+module.exports = function (grunt) {
```

Comment:
Please properly apply code-style.

Take the following things into account:

* Spaces after and before `(` and `)`.
* Usage of `'` and `"`.